domains/min.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "ducminhle25",
```

username should match github username
```suggestion
    "username": "minhducle25",
```